src/accelerate/components/List.tsx

```diff
@@ -156,33 +160,22 @@ export default function List( props: Props ) {
 					</table>
 					<div className="table-footer">
 						<div className="pagination">
-							<button
-								disabled={ page < 2 }
-								type="button"
-								onClick={ () => setPage( page - 1 ) }
-							>
-								← { __( 'Prev', 'altis' ) }
-							</button>
-							{ Array( pagination.pages ).fill( null ).map( ( _, p ) => {
-								if ( p + 1 === page ) {
-									return ( <div className="current">{ p + 1 }</div> );
-								}
-								return (
-									<button
-										type="button"
-										onClick={ () => setPage( p + 1 ) }
-									>
-										{ p + 1 }
-									</button>
-								);
-							} ) }
-							<button
-								disabled={ page <= pagination.pages }
-								type="button"
-								onClick={ () => setPage( page + 1 ) }
-							>
-								{ __( 'Next', 'altis' ) } →
-							</button>
+							{ pagination.total > 0 && (
+								<>
```

Comment:
Given that if this doesn't show then we'll have empty `.table-footer` and `.pagination` divs. We could move this logic higher so the pagination div at least is only output if we have a non zero `pagination.total` value.

You wouldn't need the fragment wrapper then.